LedgerHQ/ledger-live-desktop
# LIVE-883 LIVE-885 Support tools/swap/plugins on manager app list

## 🦒 Context (issues, jira)
https://ledgerhq.atlassian.net/browse/LIVE-883
https://ledgerhq.atlassian.net/browse/LIVE-885


## 💻  Description / Demo (image or video)
In order to continue improving the support of apps on the manager and offering specific rendering depending on the application type this PR adds a specific action for the exchange app, specific redirect for tools as well as a minor redesign of the apps table to account for the Learn More options that are now not limited to currency apps not supported in Live. This has been validated with design and we are all very happy with it as a temporary UI until we redo de manager.

<img width="1136" alt="image" src="https://user-images.githubusercontent.com/4631227/154309679-a8ad6e63-f06a-48a0-8e99-ee32af1735dd.png">
<img width="1136" alt="image" src="https://user-images.githubusercontent.com/4631227/154309786-56a3f95d-925d-4f28-9ecd-4926c9cd100c.png">



## 🖤  Expectations to reach

PR must pass CI, rebase develop if conflicts. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [x] a specific test planned is defined here
- **on delivery**: at least one of these two checkboxes must be checked: 



## How to test this
- Access the manager and look for the rendering changes for the different types of apps
- Look for installed and uninstalled versions of these types
- Supported currency apps (bitcoin) should show an Add Account CTA if installed
- Not supported currency apps (cardano) should show a Learn More CTA that redirects to the crypto assets page or specific page for currency
- Tools (recovery) should show a Learn More CTA that redirects to generic support url
- Swap apps (Exchange) should show a Swap CTA when installed that redirects to the swap feature
- Plugins (Paraswap) should show a Go to App CTA when installed that redirects to discover.
(Probably good to test in both themes)
